### PR TITLE
PLFM-6046

### PR DESCRIPTION
**Pull request by xschildw**

### CODE REVIEW

File: lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/dbo/wikiV2/V2DBOWikiPageDaoImpl.java [hunk 1]
```diff
@@ -359,8 +359,6 @@ public V2WikiPage updateWikiPage(V2WikiPage wikiPage,
 			basicDao.createBatch(attachmentsToInsert);
 		}
 		
-		// Send the change message
-		transactionalMessenger.sendMessageAfterCommit(newDbo, ChangeType.UPDATE);
```
Comment (marcomarasca) [inline lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/dbo/wikiV2/V2DBOWikiPageDaoImpl.java:363]: We should not remove this, the WIKI object type has a SNS topic in the stack builder.

File: lib/jdomodels/src/test/java/org/sagebionetworks/repo/model/dbo/dao/DBOActivityDAOImplTest.java [hunk 2]
```diff
@@ -92,7 +92,6 @@ public void testLockActivityAndIncrementEtag() throws Exception {
 		
 		ChangeType type = ChangeType.UPDATE;
 		String returnedNewEtag = activityDao.lockActivityAndGenerateEtag(id.toString(), oldEtag, type);
-		verify(mockMessenger).sendMessageAfterCommit(eq(mockDbo), eq(type));
```
Comment (marcomarasca) [inline lib/jdomodels/src/test/java/org/sagebionetworks/repo/model/dbo/dao/DBOActivityDAOImplTest.java:95]: We should not remove this, the WIKI object type has a SNS topic in the stack builder.

**Review by marcomarasca — CHANGES_REQUESTED** [review]
We should not remove the WIKI messages, but rather the ObjectType.FORUM ones (there is no SNS topic for FORUM)

File: lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/dbo/dao/DBOActivityDAOImpl.java [hunk 3]
```diff
@@ -136,7 +135,6 @@ public String lockActivityAndGenerateEtag(String id, String eTag, ChangeType cha
 		// Get a new etag
 		DBOActivity dbo = getDBO(id);
 		dbo.seteTag(UUID.randomUUID().toString());
-		transactionalMessenger.sendMessageAfterCommit(dbo, changeType);
 		return dbo.getEtag();
 	}
```
Comment (marcomarasca) [inline lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/dbo/dao/DBOActivityDAOImpl.java:136]: We should also remove the message that is sent when deleting the activity

File: lib/jdomodels/src/test/java/org/sagebionetworks/repo/model/dbo/dao/DBOActivityDAOImplTest.java (hunk 2 above)
Comment (xschildw) [reply lib/jdomodels/src/test/java/org/sagebionetworks/repo/model/dbo/dao/DBOActivityDAOImplTest.java:95]: This one still needed (activity)

Comment (marcomarasca) [reply lib/jdomodels/src/test/java/org/sagebionetworks/repo/model/dbo/dao/DBOActivityDAOImplTest.java:95]: Right, wrong class :)

**Review by marcomarasca — APPROVED** [review]

**Pull request merged**